schedule/minimal+base.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name:           minimal+base
```

Comment:
Please, exclude minimal+base. It's already done: [see minimal+base.yaml](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/schedule/minimal%2Bbase.yaml)